benchmarks/huggingface.py

```diff
@@ -354,6 +354,10 @@ def load_model(
         else:
             model = model_cls(config).to(device, dtype=dtype)
 
+        if self.args.ci:
+            # Use batch size 1 for faster CI
+            batch_size = 1
```

Comment:
Why can't we just apply this in benchmarks/common.py?

Reply:
We could, but I need to add something like an instance variable to pass around the information. I wasn't sure if your cleanup will touch this part, so I just use something quick to test the time difference.

Reply:
I guess, naively, I would have expected fiddling around with `args.batch_size` would do it, but maybe not! This is OK then